src/main/java/se/liu/ida/hefquin/cli/RunBGPOverNeo4j.java

```diff
@@ -56,6 +57,8 @@ public class RunBGPOverNeo4j extends CmdARQ
 	protected final ArgDecl argNoMerge   = new ArgDecl(ArgDecl.NoValue, "disablePathMerging");
 	protected final ArgDecl argSuppressResultPrintout = new ArgDecl(ArgDecl.NoValue, "suppressResultPrintout");
 
+	protected final String LABEL = "http://www.example.org/2000/01/rdf-schema#label";
```

Comment:
That's the wrong URI. Please use exactly the one that was/is used in `DefaultConfiguration.java`. Moreover, this member variable can also be `static` now (in addition to being `final`).